### ISSUE TITLE
Longer browser caching of ET interface.

### ISSUE THREAD

**Issue by wrdixon** [ReliefLabs/EasyTomato#46]
When remotely logging into ET, it's a ~360k to get.  That's a LOT of data to push on a slow, congested connection, making the first connection very slow.

We should change this to have a very long cache timeout to keep the loading quick when someone remotely connects.

**Comment by fougner** [comment]
Partly related, maybe use Google's CDN for jquery at least?

https://developers.google.com/speed/libraries/devguide#jquery

**Comment by djfurie** [comment]
I think that would help people remoting in from the WAN side of things, but we'd want to serve things up from the router itself if they are connecting from the LAN side.  It may be possible to determine and then we'd have a server side function like <% include_jquery_path() %> to insert the appropriate path.

**Comment by wrdixon** [comment]
I agree with djfurie.  Also we rolled our own to make it smaller.  Seems like just putting a few week cache on would do just fine.

It just kills you to wait 30+seconds on first load, every time...

**Comment by djfurie** [comment]
This is mainly a note to myself....
I think we can do the hybrid approach and host it from google as fougner suggested.  There is a function:
int get_client_info(char *mac, char *ifname)
that should return the interface that the client is logged in through.
